lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const worldStateData = require('warframe-worldstate-data');
+
+/**
+ * Translator to convert in-game names to their localizations
+ */
+class Translator {
+		
+	constructor(lang = 'en_US') {
+		this.data = worldStateData[lang];
+		if(typeof this.data !== "object") {
```

Comment:
instead default back to english if the object doesn't exist
you'd want to check if it exists before assigning, because the assignment for the data object already took place and the functions further down will end up consuming whatever it is set to.